### PR TITLE
Fixed vars for RHEL 7 to work out of the box

### PR DESCRIPTION
**Pull request by chozian**
Added necessary vars for Red Hat, and removed one which was unnecessary
since it was already defined in defaults.

### CODE REVIEW

File: meta/main.yaml [hunk 1]
```diff
@@ -40,5 +40,4 @@ galaxy_info:
     - ssh
 
 dependencies:
-  - role: geerlingguy.repo-epel
-    when: ansible_os_family == "RedHat"
```
Comment (strebitz) [inline meta/main.yaml]: Why should this dependency be changed? The new one is a fork of the old one and not using Ansible galaxy as origin. The sshguard role does not depend on a specific version of geerlingguy.repo-epel.

If you depend on a fix, please file a pull request at the upstream role.

**Review by strebitz — CHANGES_REQUESTED** [review]

File: meta/main.yaml (hunk 1 above)
Comment (chozian) [reply meta/main.yaml]: You are completely right. My bad. This is what I get for forgetting to use branches. I did not intend for this pull request to include my temporary edits to the meta dependencies. I will close this pull request and resubmit it with only the commits related to the RHEL vars. I apologize for the confusion. Thanks for your thorough response! It is helping me improve.

**Comment by chozian** [comment]
I will resubmit this pull request to include only the commits involving the RHEL vars.

**Comment by strebitz** [comment]
You could have kept this pull request open and simply squashed the obsolete commits.

File: meta/main.yaml (hunk 1 above)
Comment (chozian) [reply meta/main.yaml]: I will squash the obsolete commits for this pull request .

Comment (chozian) [reply meta/main.yaml]: Please review the most recent updates and let me know whether I did it correctly. Thanks!

**Review by strebitz — APPROVED** [review]

**Comment by strebitz** [comment]
Thank you for your contribution, @chozian !

**Pull request merged**